Executable/AudioFixer.cs

```diff
@@ -12,16 +12,20 @@ internal static void ChangeDisableUnityAudio(string path, bool newValue, Patcher
         {
```

Comment:
The `AudioFixer` will have to be moved from the `Executable` project to the `QModManager` project